lib/CoreDisTools/coredistools.cpp

```diff
@@ -317,20 +390,212 @@ size_t CorDisasm::disasmInstruction(size_t Address, const uint8_t *Bytes,
   return TotalSize;
 }
 
-void CorDisasm::printInstruction(const MCInst *MI, size_t Address,
-                                 size_t InstSize,
-                                 ArrayRef<uint8_t> Bytes) const {
-  outs() << format("%8" PRIx64 ":", Address);
-  outs() << "\t";
-  dumpBytes(Bytes.slice(0, InstSize), outs());
+void CorDisasm::printInstruction(const BlockIterator &BIter) const {
+  assert(BIter.isDecoded() && "Cannot print before Decode");
+
+  outs() << format("%8" PRIx64 ":", BIter.Addr);
```

Comment:
Is `PRIx64` portable? I thought the portable equivalent was `LLd` or something similar.